rpa-builder/6.2/modules/ROOT/pages/toolbox-variable-handling-string-operations.adoc

```diff
@@ -2,94 +2,124 @@
+image:string-operations-wizard.png[75%, 75%, The available string operations in the String Operations wizard.] 
+
+* *String operation* 
++
+The string operation you want to perform. The following operations are available:
+
+** *Analyse*
+** *Find and replace text* 
++
+Searches and replaces text found in the input text. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Find what* = `TEST`
+*** *Replace with* = `NEW`
+*** Output = `MyNEWString`
+** *Left (take first characters)*
++
+Outputs the specified number of characters from the string, starting from the first character. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Number of characters* = `4`
+*** Output = `MyTE` 
+** *Right (take last characters)*
++
+Outputs the specified number of characters from the string, starting from the last character. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Number of characters* = `4`
+*** Output = `ring` 
+** *Trim left (remove first characters)*
++
+Removes the specified number of characters from the string, starting from the first character. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Number of characters* = `3`
+*** Output = `ESTString` 
+** *Trim right (remove last characters)*
++
+Removes the specified number of characters from the string, starting from the last character. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Number of characters* = `3`
+*** Output = `MyTESTStr` 
+** *Takes first characters of the string until a specific character (specific character is not part of output)*
++
+Outputs all characters between the beginning of the string and the first occurrence of the *Character to trim at*. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Character to trim at* = `t`
+*** Output = `MyTESTS` 
++
+Note that the output string does not include the search character.
+** *Takes last characters of the string until a specific character (specific character is not part of output)*
++
+Outputs all characters between the end of the string and the first occurrence of the *Character to trim at*. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Character to trim at* = `T`
+*** Output = `String`
++
+Note that the output string does not include the search character.
+** *Removes first characters of the string until a specific character (specific character is not part of output)*
++
+Removes all characters between the beginning of the string and the first occurrence of the *Character to trim at*. For example: 
++
+*** *Input text (string)* = `MyTESTString`
+*** *Character to trim at* = `T`
+*** Output = `ESTString` 
++
+Note that the search character is removed from the output string. 
+** *Removes last characters of the string until a specific character (specific character is not part of output)*
++
+Removes all characters between the end of the string and the first occurrence of the *Character to trim at*. For example: 

```

Comment:
**issue**: Outputs all characters between the *last* occurrence of the character to trim at and the end of the string.

(if seen from the back it would be the first occurrence 😄)